tests/run/i1503.scala

```diff
@@ -0,0 +1,7 @@
+object Test {
```

Comment:
Agreed, was planning on cleaning up this PR this morning - thanks for having a look already :)